### ISSUE TITLE
Toaster component issue

### ISSUE THREAD

**Issue by Shashank03200** [shadcn-ui/ui#1077]
I am trying to use toast component in a project.
But i am getting this error 
Before this error there was another error that useCallback is not exported from React.
I configured the component.json file to fix issue but not getting the following error:
This is only with the toast component for now. Other components such as Button are working fine.

![image](https://github.com/shadcn-ui/ui/assets/55805726/bb1afd24-50c4-4f25-8bb6-c372deaf92f2)

Stack Trace: 
./node_modules/@radix-ui/react-collection/node_modules/@radix-ui/react-slot/dist/index.mjs
./node_modules/@radix-ui/react-collection/dist/index.mjs
./node_modules/@radix-ui/react-toast/dist/index.mjs
./src/src/components/ui/toast.tsx
./src/src/components/ui/toaster.tsx

**Comment by Shashank03200** [comment]
Project :  https://github.com/Shashank03200/next-reddit

**Comment by cerimorse** [comment]
Hey @Shashank03200, 

I just cloned your project locally, ran `yarn install` and then `yarn dev` and everything seemed to run okay for me. I did have to change the location of the import of the `useToast` hook in the Toaster component from `import { useToast } from "@/src/components/ui/use-toast"` to `import { useToast } from "@/components/ui/use-toast"` to match the import in the UserAuthForm component.

Maybe try doing a fresh install of your project dependencies and seeing if that works. Probably also worth sticking to a consistent component folder structure (you have some in `/src/components/ui` and some in `/src/src/components/ui`).
